src/utils/generate_sketch.py

```diff
@@ -13,13 +13,17 @@ def generate_sketch(file_path, search_db, paired_end=False):
     # first thing we want to do is query for the k-mer size.
     resp = requests.get('https://homology.kbase.us/namespace/{}/'.format(search_db))
     json_resp = resp.json()
-    if 'kmersize' in json_resp:
+    if 'sketchsize' in json_resp and json_resp['sketchsize'] is not None:
+        s = str(json_resp['sketchsize'])
+    else:
+        s = '10000'
+    if 'kmersize' in json_resp and json_resp['kmersize'] is not None:
```

Comment:
We should just do `if json_resp.get('kmersize'): ... `
This returns false if the key is not present in the dict, or if the val is None, or if the val is empty list